js/components/common/utils.js

```diff
@@ -6,7 +6,8 @@
 export function getCourse(courseName) {
   "use strict"
 
-  return fetch("course?name=" + courseName)
+  // look for courseName in database to whether add H1 or Y1 to the end
```

Comment:
Revert this change (as I discussed earlier, I fixed this problem in a separate pull request)

Reply:
You should remove this comment as well.